network/us-east-1/client-vpn/README.md

```diff
@@ -0,0 +1,132 @@
+# AWS Client VPN
+
+This layer deploy the AWS Client VPN service using AWS SSO to authenticate users.
+
+## Configure AWS SSO
+
+Due to API limitations, the SSO application must be configured manually (For more information regarding this limitation you can check [this bug](https://github.com/hashicorp/terraform-provider-aws/issues/34813) and the [notes in the AWS Terraform Provider](https://registry.terraform.io/providers/hashicorp/aws/5.39.1/docs/resources/ssoadmin_application?lang=typescript)).
+
+1. In the AWs Identity Center go to to “Applications”, then click the “Add a new application” button, then click “Add a custom SAML 2.0 application”.
+2. Complete the Name and description (if you will import this resource to terrraform note than you ewill not be able to change the name).
+3. Download the AWS SSO Metadata file (with name `saml-metadata.xml` in this folder, it will be ignored by git for of security)
+4. Enter `http://127.0.0.1:35001` as the Application ACS URL and `urn:amazon:webservices:clientvpn` as the Application SAML audience.
+5. Set up the following attributes mappings: the `Subject` needs to have the value of `${user:email}` and the format `emailAddress`. The `memberOf` attribute has a value of `${user:groups`} and should have the format `unspecified`.
+
+You can read a more complete guide in [this AWs Post](https://aws.amazon.com/es/blogs/networking-and-content-delivery/using-aws-sso-with-aws-client-vpn-for-authentication-and-authorization/). Note that some of the steps are done automatically using Terraform.
+
+### SSO Layer in Leverage
+
+To configure the group assignment as code, you can import the SSO Application in `management/global/sso`. 
+
+```bash
+leverage tf import 'aws_ssoadmin_application.client_vpn[0]' application-arn
+```
+
+And configure this section in locals.tf:
+
+```hcl
+locals{
+  enable_sso_client_vpn = true
+
+  client_vpn_groups = [
+    "devops"
+  ]
+}
+```
+
+ Alternativelly, you can use a local or variable to pass the application arn to the group assignemt resource
+
+ ```hcl
+resource "aws_ssoadmin_application_assignment" "client_vpn" {
+  for_each = local.enable_sso_client_vpn == true ? toset(local.client_vpn_groups) : toset([])
+
+  application_arn = local.aws_vpn_sso_application_arn # <<- Here
+  principal_id    = split("/", aws_identitystore_group.default[each.key].id)[1]
+  principal_type  = "GROUP"
+}
+ ```
+
+ ## Configure VPN
+
+ ### Routes
+
+It is posible to ad a route to 0.0.0.0/0 in the VPN. However, this will break the split tunnel funtionality, and may cause problems with the authorizations.
```

Comment:
```suggestion
It is posible to add a route to 0.0.0.0/0 in the VPN. However, this will break the split tunnel funtionality, and may cause problems with the authorizations.
```